src/middleware/log_request.rs

```diff
@@ -206,7 +206,19 @@ impl Display for RequestLine<'_> {
         line.add_field("at", at)?;
         line.add_field("method", self.req.method())?;
         line.add_quoted_field("path", FullPath(self.req))?;
-        line.add_field("request_id", request_header(self.req, "x-request-id"))?;
+
+        // The request_id is not logged for successful download requests
+        if !(self.req.path().ends_with("/download")
```

Comment:
Hmm, instead of special-casing here, could we set a boolean to true in the download endpoint to say "make this log line shorter"?